lib/internal/Magento/Framework/View/Element/Html/Date.php

```diff
@@ -76,7 +80,15 @@ protected function _toHtml()
     public function getEscapedValue()
     {
         if ($this->getFormat() && $this->getValue()) {
-            return strftime($this->getFormat(), strtotime($this->getValue()));
+            $localeResolver = ObjectManager::getInstance()->get(ResolverInterface::class);
```

Comment:
This needs to be  covered by tests to ensure that method output remains the same after changes.
This method expected `$this->getFormat()` to be in a form compatible with `strftime()`. That format seems to be different from what is expected\used  by `$dateFormatter->setPattern();`

Reply:
Hello @xmav, I added a converter for the date format, could you check if we can use it.

Thank you

Reply:
I added a converter for the date format, could you check if we can use it.

Thank you